easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.0.9-GCCcore-6.4.0.eb

```diff
@@ -31,6 +31,10 @@ builddependencies = [
     ('binutils', '2.28'),
 ]
 
+dependencies = [
+    ('libjpeg-turbo', '1.5.2')
```

Comment:
tests fail because a conflict was introduced here, this should be `1.5.3` instead

fixed with https://github.com/sara-nl/easybuild-easyconfigs/pull/4